src/applications/check-in/components/tests/AppointmentBlock.unit.spec.jsx

```diff
@@ -31,121 +31,236 @@ const appointments = [
-          .getAllByTestId('appointment-list-item')[0]
-          .querySelector('[data-testid="clinic-location"]'),
-      ).to.not.exist;
-      expect(
-        screen
-          .getAllByTestId('appointment-list-item')[1]
-          .querySelector('[data-testid="clinic-location"]'),
-      ).to.not.exist;
-    });
-
-    it('check in button passes axeCheck', () => {
-      axeCheck(<AppointmentBlock appointments={appointments} />);
+    describe('Phone appointment context', () => {
+      const phoneAppointments = JSON.parse(JSON.stringify(appointments));
+      phoneAppointments[0].kind = 'phone';
+      phoneAppointments[1].kind = 'phone';
+      it('Renders appointment type label', () => {
+        const screen = render(
+          <I18nextProvider i18n={i18n}>
+            <AppointmentBlock
+              appointments={phoneAppointments}
+              page="confirmation"
+            />
+          </I18nextProvider>,
+        );
+        expect(screen.getAllByTestId('appointment-type-label')[0]).to.have.text(
+          'Phone Call',
+        );
+      });
+      it('Renders appointment time with no clinic for phone appointments', () => {
+        const screen = render(
+          <I18nextProvider i18n={i18n}>
+            <AppointmentBlock
+              appointments={phoneAppointments}
+              page="confirmation"
+            />
+          </I18nextProvider>,
+        );
+        expect(screen.getByTestId('appointment-day-location')).to.have.text(
+          'Your appointments are on November 16, 2021.',
+        );
+      });
+      it('Renders phone message with appointment on confirmation page', () => {
+        const screen = render(
+          <I18nextProvider i18n={i18n}>
+            <AppointmentBlock
+              appointments={phoneAppointments}
+              page="confirmation"
+            />
+          </I18nextProvider>,
+        );
+        expect(screen.getAllByTestId('appointment-message')[0]).to.have.text(
+          'Your provider will call you at your appointment time. You may need to wait about 15 minutes for their call. Thanks for your patience.',
+        );
+      });
+      it('Renders phone message with appointment on intro page', () => {
+        const screen = render(
+          <I18nextProvider i18n={i18n}>
+            <AppointmentBlock appointments={phoneAppointments} page="intro" />
+          </I18nextProvider>,
+        );
+        expect(screen.getAllByTestId('appointment-message')[0]).to.have.text(
+          'Your provider will call you.',
+        );
+      });
+      it('Does not render facility name', () => {
+        const screen = render(
+          <I18nextProvider i18n={i18n}>
+            <AppointmentBlock appointments={phoneAppointments} page="intro" />
+          </I18nextProvider>,
+        );
+        expect(screen.queryByTestId('facility-name')).to.not.exist;
+      });
+      it('should render the appointment location for in person appointments when available', () => {
```

Comment:
this test should probably be moved out of the phone context block.

Reply:
I moved it to the in person context